modules/reindex/src/main/java/org/elasticsearch/index/reindex/RestReindexAction.java

```diff
@@ -67,11 +67,12 @@ public RestChannelConsumer prepareRequest(RestRequest request, NodeClient client
             return doPrepareRequest(request, client, true, true);
         }
 
+        boolean resilient = request.paramAsBoolean("resilient", true);
```

Comment:
I believe we settled on making this a cluster setting and not a parameter?

We can add that in a follow-up, but I think we should change this to then read a system property instead until then?

Reply:
I thought I remember that we decided to make it on a per-request basis. Anyway I changed it to a system property for now. 

Long term I think a system property == bad since it is node dependent. Sorting seems bad to me since it is super implicit. And cluster setting seems bad to me since there might be valid reasons why you want to faster performance in some circumstances.